Dockerfile

```diff
@@ -29,25 +28,23 @@ ARG DEBIAN_FRONTEND="noninteractive"
 ARG DEBCONF_NOWARNINGS="yes"
 
 # install system dependencies
-ARG SYS_DEPS="\
-    git \
-    libcurl4-openssl-dev \
-    libssl-dev \
-    openssh-client \
-    wget \
-    "
```

Comment:
I kinda prefer separating the commands from the "values"... makes it bit easier to maintain because you can really focus on what you're reviewing/modifying and not worry about syntax/context when you don't need to, but not critical.

Reply:
My thought here is that "baking in" the arguments to `apt-get` more explicitly defines the system dependencies. If you want to change one, you have to alter the dockerfile, rather than doing a one-off build with `docker build --arg`.

Overall I see sysdeps as something that we should not be changing frequently, and when we do, it should be explicit.

Reply:
Also, the ARG syntax prevent hadolint from recognizing them as packages (also not critical, but it's good to have that check)